Description: Add Mroonga disable switch
 It seems Mroonga does not build or at least does not pass
 it's own build time tests on i386. On arch amd64 it seems
 to work OK. Added a switch in the CMakeLists.txt
 so that Mroonga can be disabled in debian/rules for non-amd64.
Author: Otto Kekäläinen <otto@seravo.fi>
Bug: https://mariadb.atlassian.net/browse/MDEV-7210

--- mariadb-10.0.orig/storage/mroonga/CMakeLists.txt
+++ mariadb-10.0/storage/mroonga/CMakeLists.txt
@@ -17,6 +17,10 @@
 # License along with this library; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
+if(WITHOUT_MROONGA)
+  return()
+endif()
+
 cmake_minimum_required(VERSION 2.6)
 project(mroonga)
 
